packages/app-bridge/src/types/Terrific.ts

```diff
@@ -59,3 +59,8 @@ export type AssetChooserOptions = {
           selectedValueIds?: (number | string)[];
       }
 );
+
+export type AssetViewerOptions = {
+    token: string;
+    isDownloadable?: boolean;
```

Comment:
Could this be merged with the recently introduced `isDownloadProtected`? Or do the two values differ in some other way?

Reply:
They differ the `isDownloadProtected` is on the asset level, here it can also be protected on the guideline level or in the block itself